gr-digital/python/digital/generic_mod_demod.py

```diff
@@ -143,13 +146,21 @@ def __init__(self, constellation,
         self.rrc_filter = filter.pfb_arb_resampler_ccf(self._samples_per_symbol,
                                                        self.rrc_taps)
 
+        # Remove the filter transient at the beginning of the transmission
+        if truncate:
+            self.skiphead = blocks.skiphead(gr.sizeof_gr_complex*1, 2*11*int(self._samples_per_symbol)-1)
```

Comment:
Where does 2*11 come from?

Reply:
...can you please add a comment.

Reply:
that is an incorrect calculation of the delay caused by the pfb resampler.  need to redo the math, as it is not coming out what i expect it to be for various sample per symbol settings